labs/Certified_Anonymous_Identities-Fabric.md

```diff
@@ -0,0 +1,14 @@
+# Certified Anonymous Identities in Hyperledger Fabric
```

Comment:
Template should be followed:
```
Lab Name

Enter the name of your lab here.

Short Description

Provide a short description of your lab. This will be used for the Github repository's description.
```

Reply:
Also, the lab name should match the MD file name and it is what will be used for the repo name. The current heading looks like it could be the short description.

Reply:
@tkuhrt Thanks. We follow the template and match the MD file with the name of the lab.